src/wasm-emscripten.h

```diff
@@ -66,9 +66,12 @@ class EmscriptenGlueGenerator {
   Global* getStackPointerGlobal();
   Expression* generateLoadStackPointer();
   Expression* generateStoreStackPointer(Expression* value);
+  void generateDynCallThunk(std::string sig);
   void generateStackSaveFunction();
   void generateStackAllocFunction();
   void generateStackRestoreFunction();
+
+  std::unordered_set<std::string> sigs;
```

Comment:
maybe add a comment here saying this is all the signatures seen so far and that we've generated dynCalls for?